cmd/cosign/cli/util.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2021 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cli
+
+import (
+	"os"
+	"strconv"
+)
+
+const (
+	ExperimentalEnv = "COSIGN_EXPERIMENTAL"
+	ServerEnv       = "REKOR_SERVER"
+	rekorServer     = "https://rekor.sigstore.dev"
+)
+
+func EnableExperimentlal() bool {
```

Comment:
nit: typo :)

Reply:
gosh dangit @jonjohnsonjr droppin the ball

Reply:
if you want me to catch your typos you have to put me as a reviewer

Reply:
I thought you could just hear them, like the world's worst superpower